scvi/lightning/_trainingplans.py

```diff
@@ -100,10 +100,6 @@ def forward(self, *args, **kwargs):
         return self.module(*args, **kwargs)
 
     def training_step(self, batch, batch_idx, optimizer_idx=0):
-        # do not remove, skips over small minibatches
-        if batch[_CONSTANTS.X_KEY].shape[0] < 3:
-            return None
-
```

Comment:
@galenxing  I think there are more instances of this in the other training plans